packages/material-ui/src/utils/useControlled.d.ts

```diff
@@ -1,16 +1,23 @@
-export interface UseControlledProps {
+export interface UseControlledProps<T = any> {
```

Comment:
```suggestion
export interface UseControlledProps<T = unknown> {
```

We don't use `any` for new types anymore since it's not type-safe by default